finalPrep/modZeroHero.js

```diff
@@ -55,30 +75,72 @@ var excuse = 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.
 //   - citiesDestroyed (array)
 //   - luckyNumbers (array)
 //   - address (object with following key/values: number , street , state, zip)
-
-
+var scaryMonster = {
+  name: 'The Flying Purple People Eater',
+  smell: 'Oddly, not that bad',
+  weight: 10,
+  citiesDestroyed: ['Tokyo','Sydney'],
+  luckyNumbers: [3,6,14],
+  address: {
+    number: 123,
+    street: 'Main',
+    state: 'New York',
+    zip: 12345,
+  }
+}
+console.log(scaryMonster);
 // Create a new class called SuperHero
```

Comment:
This whole SuperHero section is 💯! You are in a very strong place heading into Mod 1, even if you had to look some things up, etc.